astropy/units/tests/test_quantity_array_methods.py

```diff
@@ -1,11 +1,15 @@
 # The purpose of these tests are to ensure that calling quantities using
 # array methods returns quantities with the right units, or raises exceptions.
+import os
 
 import pytest
 import numpy as np
 from numpy.testing import assert_array_equal
 
 from astropy import units as u
+from astropy.utils.compat import NUMPY_LT_1_22
+
+IS_S390X = os.environ.get('ARCH_ON_CI', 'normal') == 's390x'
```

Comment:
Must be this line...

Reply:
Why is the `xfail` not working? 🤯 

`ARCH_ON_CI: s390x`
`Numpy: 1.20.3`